packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -196,6 +200,21 @@ export function publish(options: InitOptions): void {
       const topFlowlet = flowletManager.top();
       let flowlet = topFlowlet ?? defaultTopFlowlet; // We want to ensure flowlet is always assigned
       if (shouldPushPopFlowlet(event)) {
+        let groupFlowlet;
+        if (event.target != null) {
+          const targetElement = event.target;
+          if (eventName === 'mouseover') {
+            const type = UIEventGroup.MOUSE;
+            groupFlowlet = new flowletManager.flowletCtor(`${type}(element:${autoLoggingID})`);
+            groupFlowlets.set(targetElement, {groupFlowlet, type});
+          } else if (eventName === 'mouseout' || eventName === 'click') {
+            const groupFlowletInfo = groupFlowlets.get(targetElement);
+            if (groupFlowletInfo != null && groupFlowletInfo.type === UIEventGroup.MOUSE) {
+              groupFlowlet = groupFlowletInfo.groupFlowlet;
+            }
+          }
+        }
```

Comment:
This logic is ok, but I think a better approach is to have a static map from `eventName` to `(autoLoggingID: string)=>IALFlowlet | undefined` outside of the event handler, and then just read the `groupFlowletGetter` function outside and then run it here. 
This way, as we add more and more events to the list, it is easy to extend it and cost remains constant.

Reply:
Based on our offline conversation, I gave this a try as part of this [PR](https://github.com/facebook/hyperion/pull/138/files#diff-062df1927fb75ce28648c7b3680f678c0be0efa6d2e4c7c406a710b054550438), since I needed something to control general trigger flowlet assignment.